### PR TITLE
[15.0][MIG] account_invoice_mode_monthly (renamed)

### PR DESCRIPTION
**Pull request by TDu**
This module migration depends on #1243

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-partner_invoicing_mode @ git+https://github.com/OCA/account-invoicing@refs/pull/1243/head#subdirectory=setup/partner_invoicing_mode
```
Comment (i-vyshnevska) [inline test-requirements.txt:1]:
```suggestion
odoo15-addon-partner_invoicing_mode @ git+https://github.com/OCA/account-invoicing@refs/pull/1243/head#subdirectory=setup/partner_invoicing_mode
```

**Comment by StefanRijnhart** [comment]
Please remove the migration here as well.

**Comment by TDu** [comment]
@StefanRijnhart Done

**Review by StefanRijnhart — APPROVED** [review]
Thanks! Any chance you could review the 16.0 version of these modules? https://github.com/OCA/account-invoicing/pull/1295

**Pull request closed without merging**